Ginger/Ginger/UserControlsLib/UCListView/UcListViewItem.xaml

```diff
@@ -156,7 +156,7 @@
                         <!--to set width per need like for notifications-->
                     </Grid.ColumnDefinitions>
 
-                    <TextBlock x:Name="xItemDescriptionTxtBlock" Grid.Column="0" Text="Description" FontSize="11" Margin="2 0 0 0" Foreground="{StaticResource $BackgroundColor_DarkGray}" FontStyle="Normal" FontWeight="Normal" TextTrimming="CharacterEllipsis" TextWrapping="NoWrap" HorizontalAlignment="left" VerticalAlignment="Center" />
+                    <TextBlock x:Name="xItemDescriptionTxtBlock" Grid.Column="0" Text="Description" FontSize="11" Margin="2 0 0 0" Foreground="{StaticResource $BackgroundColor_DarkGray}" FontStyle="Normal" FontWeight="Normal" TextTrimming="CharacterEllipsis" TextWrapping="NoWrap" Width="400" HorizontalAlignment="left" VerticalAlignment="Center" />
```

Comment:
Please verify the impact of width at all places wherever it is used

Reply:
I have limited the width of the description to 400px, because it makes the card's width bigger than its usual width. I have checked in the all the list views and it works as expected and additionally if the description is longer we provide a tooltip for the user.